### ISSUE TITLE
Event or property to set dynamic panel tab title on init

### ISSUE THREAD

**Issue by nikhilpatel7** [surveyjs/survey-library#9146]
**Is your feature request related to a problem? Please describe.**
I am using to set nested dynamic panel tab title using "onGetDynamicPanelTabTitle" event on a field change and set the value.
As per your documentation :https://surveyjs.io/survey-creator/documentation/end-user-guide/dynamic-tab-title-format#dynamic-panel-title-pattern

My issue is I want to set the same title which was set using "onGetDynamicPanelTabTitle" when user re-open the survey. 
Basically in Edit mode I am not able to set the nested dynamic panel tab titles on survey load unless user changes a field value the title is not set, that is something I want to achieve without user interaction (on load). I tried few things but none works and I cannot find anything relevant which can set tab title on load. 

If you have any relevant property of event that I can use please guide me. Thanks

**Comment by JaneSjs** [comment]
Hello,
When you reopen a survey, the [`survey.onGetDynamicPanelTabTitle`](https://surveyjs.io/form-library/documentation/api-reference/survey-data-model#onGetDynamicPanelTabTitle) function is invoked to allow you to specify custom tab titles. When you open a form with existing user answers set via `survey.data`, titles will be also updated. Please refer to the following demo: [View Demo](https://plnkr.co/edit/SIQZ5nkRfvYXYkZR).

![image](https://github.com/user-attachments/assets/032f01d6-effe-4e07-8200-ad232b8b2fe3)

Let me know if it works for you.

**Comment by nikhilpatel7** [comment]
Sorry @JaneSjs for late response. I have tried what you mentioned above, I am working with Vue js 2 (type script/JS) 
 
 HTML:
```
 <template>
     <survey v-if="surveyModel" :survey="surveyModel" />
 </template>
```
 
Initialization on page mounted()
```
  const surveyJSModel = new Model(surveyOnJob.surveyContent);
     
      //if a survey response has been started apply the current answers to the survey so the user can proceed
      if(surveyOnJob.surveyData){
        surveyJSModel.data = JSON.parse(surveyOnJob.surveyData);
      }
    this.surveyModel = surveyJSModel;

 surveyJSModel.onGetDynamicPanelTabTitle.add((survey, options) => {
            var itemIndex= options.visiblePanelIndex + 1;
            const description = options.panel.getQuestionByName("DESCRIPTION").value || "Description";
            options.title = itemIndex+ '- '+ description .substring(0, 15);
            if(description .length > 15) {
              options.title += '..';
            }
      });
```
 
"**onGetDynamicPanelTabTitle**" is not invoked when I redirect user to this page.
It is invoked when an element is change/updated inside the tab panel which is what I need as well. But it does not update the title when I load survey in edit mode. Can you please guide me what I am doing wrong here? that would be great if you point out the issue. Thanks 
 
 Let me know if you need more information. I will revert as soon as possible.
